agent/src/com/jonnyzzz/teamcity/virtual/run/VMRunnerContext.java

```diff
@@ -68,14 +69,20 @@ public File getAgentTempDirectory() {
 
   @NotNull
   public String getCheckoutMountPoint() {
+    String defaultMountPoint;
+    if(SystemInfo.isUnix) {
```

Comment:
Let's check for `SystemInfo.isWindows` instead. In that case it's clear a path should be `c:\...`